requirements/dev.in

```diff
@@ -3,6 +3,6 @@
 
 appnope; sys_platform=='darwin'
 ipdb
-pip-tools
+pip-tools==5.2.1
```

Comment:
Would >=5.2.1 be better? Or do later versions not work? I can't see anything suspicious in the changelog. And 5.3 adds pip>=20.2 support.